frx_challenges/web/management/commands/evaluator.py

```diff
@@ -167,15 +166,6 @@ async def process_running_evaluation(
     async def ahandle(self):
         evaluator = DockerEvaluator()
         while True:
-            # Create evaluation objects when they do not exist
-            versions_without_evaluations = Version.objects.filter(
-                Q(status=Version.Status.UPLOADED),
-                ~Exists(Evaluation.objects.filter(version=OuterRef("pk"))),
-            )
-            async for v in versions_without_evaluations:
-                e = Evaluation(version=v)
-                await e.asave()
```

Comment:
Okay good, now that every version is strictly associated with an evaluation, there is no need to create evaluation objects when they do not exist.